CIP-????/README.md

```diff
@@ -0,0 +1,104 @@
+
+## Motivation: Why is this CIP necessary?
+
+[CIP-0027](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0027) provides great groundwork for resale fee functionality, but lacks support for updating fee rates/destination addresses, and also lacks support for multiple fee outputs. Projects may choose to maintain multiple wallets and split resale fees between them. DAOs may want to to govern their own asset collection's resale fee rates and their fee destinations. A use-case example would be a digital collectible project that wants to charge resale fees of 4% to a DAO wallet and 1% to a core team wallet. Another example would be an artist who wants to, just for the next 30 days, send all resale fees of their minted assets to a charity wallet, and decrease their policy's fee rate during that time as well. Because this new standard changes two features of [CIP-0027](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0027), it is being proposed as a new CIP rather than as an update to [CIP-0027](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0027).
+
+## Specification
+
+- A new tag of 778 is proposed for this implementation. The 778 tag compliments [CIP-0027](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0027)'s 777 tag.
+- Any unlocked policy can write initial or updated resale fees tags to a new token.
+- The resale fees tags are to be written to an unnamed token, using the policy to be used for the intended Cardano Assets.
+- The latest minted set of instructions will always be honored. This allows a Cardano Asset maker to change the resale fees at a future date.
+- Within this created asset will be the metadata for resale fees distributions.  It will use a tag of 778, and then have two or more tags to identify the percentages of future sales requested as resale fees, and the payment addresses to forward those resale fees to. The first set of two tags will be "rate" and "addr". Any additional tag sets will repeat after the first set of two tags, starting with "rate2" and "addr2", with sequentially increasing numbers for each added set.
+- The "rate*" key tags can be any floating point value from 0.0 to 1.0, to represent between 0 and 100 percent. For example, a 12.5 percent resale fee would be represented with "rate": "0.125".
+- The "addr*" key tags can be string values, or an array. It is to include a single payment address. By allowing for an array, the payment address can exceed the per line 64 character limitation. This payment address could be part of a smart contract, which should allow for greater flexibility of resale fees distributions, controlled by the asset creator.
```

Comment:
We should avoid "keyword" or "wildcard" pattern matching like this because it makes the system highly prone to human error, we should instead rely on lists of objects (see comments below)